introduction.md

```diff
@@ -0,0 +1 @@
+My name is Luke Eastman. I am 25 and live in Yellow Springs, OH
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.